docs-2.0/16.eco-tools/3.nebula-spark-connector.md

```diff
@@ -0,0 +1,35 @@
+# Nebula Spark Connector
+
+Nebula Spark Connector is a Spark connector application for reading and writing Nebula Graph data in Spark standard format. Nebula Spark Connector consists of two parts: Reader and Writer.
+
+* Reader
+  
+  Provide a Spark SQL interface. This interface can be used to read Nebula Graph data. Read one vertex or edge type data at a time, and assemble the result into a Spark DataFrame.
+
+* Writer
+
+  Provide a Spark SQL interface. Use this interface to write data in DataFrame format to Nebula Graph on a strip-by-strip or bulk basis.
+
+For more information, see [Nebula Spark Connector](https://github.com/vesoft-inc/nebula-spark-utils/blob/v2.0.0/nebula-spark-connector/README.md).
+
+## Use cases
+
+Nebula Spark Connector applicable to the following scenarios:
```

Comment:
"Nebula Spark Connector is applicable in"
or
"Nebula Spark Connector applies to"